docs/configuration.md

```diff
@@ -491,6 +495,27 @@ map.
 >are equivalent, `layerinfo` has been deprecated, in favor or
 >`blobdescriptor`.
 
+### foreignlayers
```

Comment:
This shouldn't really be called "Foreign Layers". The registry doesn't and shouldn't know what a foreign layer is.

This feature influences the validation of manifest urls.